part-spec/power.json

```diff
@@ -0,0 +1,210 @@
+{
+    "$id": "https://github.com/chromeos/digital-datasheets/blob/main/part-spec/power.json",
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "type": "object",
+    "title": "power",
+    "oneOf": [
+        {
+            "title": "switching converter",
+            "type": "object",
+            "required": [
+                "partType",
+                "manufacturer",
+                "orderableMPN",
+                "sourceDatasheetID",
+                "digitalDatasheetID"
```

Comment:
Let's discuss in the WG on the additional required properties for the Buck Converter